crates/relayer/src/chain/cosmos.rs

```diff
@@ -618,54 +631,12 @@ impl ChainEndpoint for CosmosSdkChain {
         Ok(HealthCheck::Healthy)
     }
 
-    /// Fetch a header from the chain at the given height and verify it.
```

Comment:
While I agree that the current order of definitions is subpar, I would rather not move code around in this PR to help with reviewing it and keeping the diff as small as possible.

Can you please undo these changes and keep the original ordering?

Reply:
Replaced all the methods based on cosmos.rs in the main branch.